File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/BacktrackingJava/SampleAuto.java
package org.firstinspires.ftc.teamcode.BacktrackingJava;

import com.acmerobotics.dashboard.FtcDashboard;
import com.acmerobotics.dashboard.telemetry.MultipleTelemetry;
import com.acmerobotics.roadrunner.Pose2d;
import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode;

@Autonomous(name = "SampleAuto", group = "Linear OpMode")
public class SampleAuto extends LinearOpMode {

    @Override
    public void runOpMode() throws InterruptedException {
        telemetry = new MultipleTelemetry(telemetry, FtcDashboard.getInstance().getTelemetry());
        //start Pose
        Drive drive = new Drive(hardwareMap, new Pose2d(0.0, 0.0, 0.0));

        waitForStart();
        while (opModeIsActive()) {
            drive.updatePoseEstimate();

            // Run your trajectories


            telemetry.update();
        }
    }
}

File: TeamCode/src/main/java/org/firstinspires/ftc/teamcode/BackTrackingIMU-master/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/BacktrackingKt/SampleAuto.kt
package org.firstinspires.ftc.teamcode.BacktrackingKt

import com.acmerobotics.dashboard.FtcDashboard
import com.acmerobotics.dashboard.telemetry.MultipleTelemetry
import com.acmerobotics.roadrunner.Pose2d
import com.qualcomm.robotcore.eventloop.opmode.Autonomous
import com.qualcomm.robotcore.eventloop.opmode.LinearOpMode

@Autonomous(name = "SampleAutoKt", group = "Linear OpMode")
class SampleAuto : LinearOpMode() {
    override fun runOpMode() {
        telemetry = MultipleTelemetry(telemetry, FtcDashboard.getInstance().telemetry)

        val drive = Drive(hardwareMap, Pose2d(0.0, 0.0, 0.0))

        waitForStart()

        while (opModeIsActive()) {
            drive.updatePoseEstimate()

            // Run your trajectories

            telemetry.update()
        }
    }
}
